mh/quest_utils.py

```diff
@@ -1280,10 +1280,14 @@ def make_binary_event_quest(quest_id, name, client, description, flags,
     data += b'\0' * (4 * 11)
 
     assert len(data) == 0x4B4
+    tail = 0x4B4 + 4
     try:
-        data += read_quest_sm_data(smallmonster_data)
+        sm_data = read_quest_sm_data(smallmonster_data)[4:]
+        tail += len(sm_data)
+        data += struct.pack(">I", tail)
+        data += sm_data
     except Exception as e:
-        pass
+        data += struct.pack(">I", tail)
```

Comment:
Since small monster data is directly ripped from the game, it cannot be shared in this repository. This block allows us to "program in" small monster data that we have ripped ourselves, while still allowing for no small monster data to be provided in the case of a person downloading this repository and trying to run it without going through the trouble to do so.

Reply:
As I said, this should be documented. Doing it this way looks clearly like a hack without any kind of documentation. Keep in mind that this project is open-source and someone else might be looking into the code and try to make sense of it.